packages/rulesets/src/asyncapi/functions/utils.ts

```diff
@@ -0,0 +1,20 @@
+export function parseUrlVariables(str: string): string[] | undefined {
```

Comment:
Personal preference, but I appreciate how each top level file in the `functions` folder represents a single named function. Adding a `functions/utils.ts` breaks the convention - any chance we could move these to a nested `utils` folder, similar to how it's organized in `packages/rulesets/src/oas/functions`?

Reply:
Ok 👍🏼 I moved helper functions to the `asyncapi/functions/utils` alongside with unit tests. Thanks!